include/inviwo/core/datastructures/volume/volumeramprecision.h

```diff
@@ -46,8 +46,10 @@ class VolumeRAMPrecision : public VolumeRAM {
 public:
     using type = T;
 
-    VolumeRAMPrecision(size3_t dimensions = size3_t(128, 128, 128));
-    VolumeRAMPrecision(T* data, size3_t dimensions = size3_t(128, 128, 128));
+    VolumeRAMPrecision(size3_t dimensions = size3_t(128, 128, 128),
+                       const SwizzleMask& swizzleMask = swizzlemasks::rgba);
+    VolumeRAMPrecision(T* data, size3_t dimensions = size3_t(128, 128, 128),
```

Comment:
should not have any default dims here since we hand in a pointer...

Reply:
should probably make them explicit also...

Reply:
done.